lib/td/command/export.rb

```diff
@@ -48,6 +49,10 @@ def table_export(op)
       raise ArgumentError, "#{s} is not a supported encryption method" unless SUPPORTED_ENCRYPT_METHOD.include?(s)
       encryption = s
     }
+    op.on('-a', '--assume-role SESSION_NAME#ROLE_ARN', 'export with assume role with SESSION_NAME as session name and ROLE_ARN as role arn') {|s|
+      raise ArgumentError, "#{s} is not a supported assume role style. Please add ROLE_ARN after SESSION_NAME#" unless s.include?('#')
```

Comment:
Thanks.
It seems that there is no problem to drop RoleSessionName, then I fixed to use only assume role arn.
http://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html